src/test/java/io/logz/LogzioSenderTest.java

```diff
@@ -67,11 +67,11 @@ public void simpleAppending() throws Exception {
         // Sleep double time the drain timeout
         Thread.sleep(drainTimeout * 1000 * 2);
 
-        assertTrue(mockListener.checkForLogExistence(token, type, loggerName, Level.INFO, message1, false, null));
-        assertTrue(mockListener.checkForLogExistence(token, type, loggerName, Level.WARN, message2, false, null));
+        assertTrue(mockListener.checkForLogExistence(token, type, loggerName, Level.INFO, message1, false, null, null));
```

Comment:
No lots of overrides. It's like:
f(a,b,c)

Now you want to have: f(a,b,c, optional d)
so refactor f(a,b,c) to call f(a,b,c, Optioal.empty())